aws/data_source_aws_ecs_container_definition.go

```diff
@@ -66,9 +67,11 @@ func dataSourceAwsEcsContainerDefinition() *schema.Resource {
 func dataSourceAwsEcsContainerDefinitionRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).ecsconn
 
-	desc, err := conn.DescribeTaskDefinition(&ecs.DescribeTaskDefinitionInput{
+	params := &ecs.DescribeTaskDefinitionInput{
 		TaskDefinition: aws.String(d.Get("task_definition").(string)),
-	})
+	}
+	log.Printf("[DEBUG] Reading ECS Container Definition: %#v", params)
```

Comment:
FWIW the AWS SDK struct implement `Stringer()`, so you can use `%s` here instead of `%#v`

Reply:
@catsby Made the change, thanks for letting me know! :)